File: TRANSCRIPTER-JS/change-code.js
const fs = require('fs');
const readline = require('readline');

async function processLineByLine() {
  const fileStream = fs.createReadStream('test.txt');

  const rl = readline.createInterface({
    input: fileStream,
    crlfDelay: Infinity
  });

  var newLines = [];
  for await (const line of rl) {
    let newLine = 'client.println("' + line + '");';
    newLine = newLine.replace(/\s+/, "");
    newLines.push(newLine);
    // console.log(`Line from file: ${newLine}`);
  }
  return newLines;
}

let transformedLines = processLineByLine().then(newLines => newLines);
console.log(transformedLines);

// async function run()
// {
//     const data = await processLineByLine();
//     console.log(data);
// }
// run();

File: TRANSCRIPTER-PY/change-code.py
def transformClient(file):
    transformedLines = []
    with open(file) as fp:
        line = fp.readline()
        while line:
            line = line.strip()
            line = 'client.println("' + line + '");'
            line = line.replace('\n', '')
            line+='\n'
            transformedLines.append(line)
            line = fp.readline()
    return transformedLines

def createNewFile(newFileParam):
    finalTxtFile = open("finalTxtFile.txt", "w+")
    for line in range(len(newFileParam)):
        finalTxtFile.write(newFileParam[line]+'\n')

createNewFile(transformClient('test.txt'))